controllers/replicationsource.go

```diff
@@ -154,7 +156,7 @@ func (r *VolumeSnapshotBackupReconciler) setVSBRepSourceStatus(log logr.Logger)
 			return true, nil
 
 			// ReplicationSource phase is still in progress
-		} else if !repSourceCompleted && reconConditionProgress.Type == volsyncv1alpha1.ConditionSynchronizing {
+		} else if !repSourceCompleted && reconConditionReason.Reason == volsyncv1alpha1.SynchronizingReasonSync {
```

Comment:
For 0.5 volsync are preserving condition type `synchronizing` right ? then why are we basing this on reason ? Any particular update ?

Reply:
See above comment